### PR TITLE
Removes Xeno egg spawn chance from metastation

### PR DESCRIPTION
**Pull request by Niknakflak**
Apparently this works, here you go coiax

### CODE REVIEW

**Comment by coiax** [comment]
no, that's fine

**Comment by Kromgar** [comment]
Not enough xenos

**Comment by KorPhaeron** [comment]
Why?

**Comment by Cyberboss** [comment]
See #25285

**Comment by WJohn** [comment]
To nobody's surprise, I'm _not_ okay with this 👎

**Comment by Niknakflak** [comment]
coiax told me to, and it looks like I have oranges' support.  Also, who cares about wjohn's opinion

**Comment by Militaires** [comment]
@Niknakflak kor

**Comment by Niknakflak** [comment]
Is my answer not satisfactory for you bman

**Comment by Militaires** [comment]
maybe

**Comment by GunHog** [comment]
Reading the diff reveals no change to the egg spawner. It is a deceptive title meant to provoke people. (We really need to crack down on this behavior.)

**Comment by Iamgoofball** [comment]
local man annoyed by fetish removal

more at 11

**Comment by WJohn** [comment]
didn't think niknak would be the kind of guy to make a _fake_ PR just to provoke reactions. i thought it'd at least be real.

**Comment by KorPhaeron** [comment]
Someone who isnt a maintainer telling you to do something is not rationale for how a change would improve the game

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -128703,7 +128703,7 @@ cTD
 cRi
 cSn
 daF
-daJ
+cSn
```
Comment (Iamgoofball) [inline _maps/map_files/MetaStation/MetaStation.dmm]: well something gets done here so

Comment (Iamgoofball) [inline _maps/map_files/MetaStation/MetaStation.dmm]: yeah this actually does remove the spawner according to the key values

Comment (GunHog) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Oh, well. Horrible. (I cannot read maps well)

**Pull request closed without merging**